content/en/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -149,7 +146,7 @@ descriptions on this page:
 * [Bayesian optimization](#bayesian)
 * [HYPERBAND](#hyperband)
 * [Hyperopt TPE](#tpe-search)
-* [NAS based on reinforcement learning](#nas)
+* [Efficient Neural Architecture Search (ENAS)](#enas)
```

Comment:
I think we are not using ENAS, we do not share parameters. Our approach is part of ENAS.

Reply:
@gaocegege Should we name it Neural Architecture Search based on ENAS. 
What do you think?

Reply:
SGTM. We can say this algorithm is ENAS-based

Reply:
Done.